core/src/main/java/com/adobe/aem/dot/dispatcher/core/DispatcherConfigurationFactory.java

```diff
@@ -60,20 +61,22 @@
     if (StringUtils.isEmpty(repoPath)) {
       throw new IllegalArgumentException("The repo folder is not set.");
     }
-    if (StringUtils.isEmpty(dispatcherAnyFilePath)) {
-      logger.info("The Dispatcher configuration directory is not set.");
-    }
 
     // first check to see if the repo that we're given exists (the mount point)
     File repoFile = new File(repoPath);
-
     if (!repoFile.exists()) {
       throw new ConfigurationException(MessageFormat.format("The `{0}` folder does not exist", repoPath));
     }
 
-    // now we can see if the 'anyDir' exists, this directory contains the dispatcher.any file
-    File dispatcherAnyFile = FileUtils.getFile(repoPath, dispatcherAnyFilePath + File.separator +
-                                                                 DispatcherConstants.DISPATCHER_ANY);
+    File dispatcherAnyFile;
```

Comment:
This is the change from last week - do not try to use `dispatcherAnyFilePath` if it is null.  (There was a null in the path.)